src/audio/multiband_drc/tune/multiband_drc_paths.m

```diff
@@ -0,0 +1,33 @@
+function multiband_drc_paths(enable)
+
+% multiband_drc_paths(enable)
+% enable - set to true to enable needed search path
+%          set to false to disable the search paths
+%
+
+% SPDX-License-Identifier: BSD-3-Clause
+%
+% Copyright (c) 2024, Intel Corporation. All rights reserved.
+
+	common = '../../../../tools/tune/common';
+	crossover = '../../../../tools/tune/crossover';
+	drc = '../../../../tools/tune/drc';
+	eq = '../../../../tools/tune/eq';
+	# After #9215 merge use this:
+	# crossover = '../../crossover/tune';
+	# After #9188 merge use this:
+	# drc = '../../drc/tune';
+	# After #9187 merge use this:
+	# eq = '../../eq_iir/tune';
```

Comment:
Btw, does Matlab/Octave support PATHs via environment variable ? If so, might be easiest to have a script that sets these for the (like Zephyr has convenience script to setup build environment)

Reply:
Yes, the environment variables can be supported:

```
octave:1> getenv('SOF_WORKSPACE')
ans = /home/singalsu/work/current/sof
```

Would above be better e.g. as `tune = sprintf('%s/sof/tools/tune', getenv('SOF_WORKSPACE'));` ?

Reply:
Btw, lets track this as a long term feature - ideally long term west would enable this when we configure the environment.